ordered logistic regression.ipynb

```diff
@@ -16,34 +16,50 @@
    "outputs": [],
```

Comment:
<p>Let's leave this as it was</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MarcoGorelli/pydataglobal-21/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>